index.html

```diff
@@ -0,0 +1,230 @@
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <div class="cv">
+      <div class="container">
+        <aside class="left_side">
+          <div class="profileText">
+            <div class="imgBx">
+              <img src="face.svg" alt="Profile image" />
+            </div>
+            <h2>Naufil Shemle<br /><span>Web Developer</span></h2>
+          </div>
+
+          <section class="contactInfo">
+            <h3 class="title">Contact Info</h3>
+            <ul>
+              <li>
+                <span class="icon"
+                  ><i class="fa fa-phone" aria-hidden="true"></i
+                ></span>
+                <span class="text">+91 666 666 6666</span>
+              </li>
+              <li>
+                <span class="icon"
+                  ><i class="fa fa-envelope-o" aria-hidden="true"></i
+                ></span>
+                <span class="text">naufil.dev@gmail.com</span>
+              </li>
+              <li>
+                <span class="icon"
+                  ><i class="fa fa-linkedin" aria-hidden="true"></i>
+                </span>
+                <span class="text">www.linkedin.com/me</span>
+              </li>
+              <li>
+                <span class="icon"
+                  ><i class="fa fa-map-marker" aria-hidden="true"></i
+                ></span>
+                <span class="text">Mumbai, India</span>
+              </li>
+            </ul>
+          </section>
+
+          <section class="contactInfo education">
+            <h3 class="title">EDUCATION</h3>
+            <ul>
+              <li>
+                <h5>2021 - Present</h5>
+                <h4>Bachelor Degree in Computer</h4>
+                <h4>Mumbai University</h4>
+              </li>
+              <li>
+                <h5>2018 - 2021</h5>
+                <h4>Diploma in IT</h4>
+                <h4>MSBTE</h4>
+              </li>
+            </ul>
+          </section>
+
+          <section class="contactInfo lang">
+            <h3 class="title">Languages</h3>
+            <ul>
+              <li>
+                <span class="text">Urdu</span>
+                <span class="percent">
+                  <div style="width: 90%"></div>
+                </span>
+              </li>
+              <li>
+                <span class="text">English</span>
+                <span class="percent">
+                  <div style="width: 80%"></div>
+                </span>
+              </li>
+              <li>
+                <span class="text">Hindi</span>
+                <span class="percent">
+                  <div style="width: 85%"></div>
+                </span>
+              </li>
+              <li>
+                <span class="text">Marathi</span>
+                <span class="percent">
+                  <div style="width: 80%"></div>
+                </span>
+              </li>
+            </ul>
+          </section>
```

Comment:
```suggestion
          <section class="contactInfo lang">
            <h3 class="title">Languages</h3>
            <ul>
              <li>
                <span class="text">Urdu</span>
                <div class="percent">
                  <div style="width: 90%"></div>
                </div>
              </li>
              <li>
                <span class="text">English</span>
                <div class="percent">
                  <div style="width: 80%"></div>
                </div>
              </li>
              <li>
                <span class="text">Hindi</span>
                <div class="percent">
                  <div style="width: 85%"></div>
                </div>
              </li>
              <li>
                <span class="text">Marathi</span>
                <div class="percent">
                  <div style="width: 80%"></div>
                </div>
              </li>
            </ul>
          </section>
```

I think you shouldn't nest a block-level element inside an inline element...

(block-level elements may contain inline or block) and
(inline may contain inline only)